litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Line } from 'rc-progress';
+import React, { useEffect, useState } from 'react';
+
+interface LinearProgressBarProps {
+  value: number;
+  maxValue: number;
+}
+
+const AnalyticsLinearProgressBar: React.FC<LinearProgressBarProps> = ({
+  value,
+  maxValue,
+}) => {
+  const [color, setColor] = useState(' ');
+  const width: number = 2;
+  const resultValue = ((value as number) / (maxValue as number)) * 100;
+  useEffect(() => {
+    return setColor('#5B44BA');
```

Comment:
why does color require to be a state variable if I never changes.
use a fixed color from theme data with useTheme()

Reply:
Some components were already there just reused code from them, will make changes for this.. @arkajyotiMukherjee